src/js/api/errors.js

```diff
@@ -39,6 +39,21 @@ export const SETUP_ERROR_LOADING_PLAYLIST = 102000;
  */
 export const ERROR_LOADING_PLAYLIST = 202000;
 
+/**
+ * @enum {ErrorCode} Setup failed because the initial provider failed to load
+ */
+export const SETUP_ERROR_LOADING_PROVIDER = 104000;
+
+/**
+ * @enum {ErrorCode} Between playlist items, the required provider could not be loaded be
+ */
+export const PLAYLIST_ERROR_LOADING_PROVIDER = 203000;
```

Comment:
The `203,000` category is for Playlist items, so we should rename this to `ERROR_LOADING_PLAYLIST_ITEM`. The category covers more than provider errors, i.e. we can have code 203,640 which is the `No media` error.

Reply:
Also let's add periods to the descriptions, it seems to be the pattern in api.js